docs/development/bump-node-major.md

```diff
@@ -0,0 +1,20 @@
+# How to bump Renovate to next NodeJS LTS release
```

Comment:
Note for later: we should add some sample PRs to this documentation once we have some examples. It's a process that I expect we'll go through once every 6 months, so having examples always helps.

Reply:
maybe just link the existing merged pr's?

Reply:
Yeah, but we don't have any for this new `node-next` workflow. I am happy to wait until Node v16 happens 😄